HW3-Spark.py

```diff
@@ -0,0 +1,92 @@
+# HW3 - Spark Assignment
+# BDA 696 -  Karenina Zaballa
+
```

Comment:
I ran your code and your first two graphs poped up but not the third. Please review your code as I think it has something to do with your data table columns per the error I am receiving. Error Below: 

pyspark.sql.utils.AnalysisException: cannot resolve 'date_sub(CAST(ba1.`local_date` AS DATE), INTERVAL '100 days')' due to data type mismatch: argument 2 requires (int or smallint or tinyint) type, however, 'INTERVAL '100 days'' is of interval type.; line 1 pos 247;
'Aggregate ['ba1.game_id, 'ba1.batter, 'ba1.local_date], ['ba1.batter, 'ba1.game_id, 'SUM('ba2.Hit) AS Hit_Sum#645, 'SUM('ba2.atBat) AS AtBat_Sum#646, 'ba2.local_date]
+- 'Join Inner, (((batter#585 = batter#648) AND (local_date#651 > date_sub(cast(local_date#588 as date), 100 days))) AND (local_date#588 > local_date#651))
   :- SubqueryAlias ba1
   :  +- SubqueryAlias ba_temp
   :     +- Relation[game_id#584,batter#585,Hit#586,atBat#587,local_date#588] JDBCRelation((SELECT game.game_id, batter_counts.batter,                     batter_counts.Hit, batter_counts.atBat, game.local_date                     FROM batter_counts                    JOIN game ON batter_counts.game_id = game.game_id)ba_temp) [numPartitions=1]
   +- SubqueryAlias ba2
      +- SubqueryAlias ba_temp
         +- Relation[game_id#647,batter#648,Hit#649,atBat#650,local_date#651] JDBCRelation((SELECT game.game_id, batter_counts.batter,

Reply:
Super helpful. Thank you! My connection is out of whack currently. Might need a screenshot of the graphs.

Reply:
Hint: If you want to put code in a spot. You can do something like the following:

I have three ` in front and behind the statement:
```
Code goes here
```